unity-renderer/Assets/UIComponents/Scripts/Components/ColorPicker/ColorSelector.cs

```diff
@@ -30,7 +30,13 @@ public void Populate(IReadOnlyList<Color> colors)
     public void Cleanup()
     {
         foreach (var colorToggle in colorToggles)
+        {
+#if UNITY_EDITOR
+            DestroyImmediate(colorToggle.gameObject);
+#else
```

Comment:
You also need to check for `Application.IsPlaying()`. You can only use `DestroyImmediate` when unity is not in play mode.

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/5136/commits/7cc075f8b1d1fb8199b23da4d5bc37de98cbaf03

Thanks!